### DISCUSSION TITLE
OS poll

### DISCUSSION THREAD

**Discussion by ghishadow** [lapce/lapce#1026] [Polls]
just for research purposes

**Comment by panekj** [comment]
>*doesn't have "all of the above" option*  
  
\>:(

**Reply by ghishadow** [reply]
added 👍🏻

**Comment by theAkito** [comment]
It would've just been better to add the option to choose 1 or more, instead of "all of the above", which is an extremely rare case.

**Reply by ghishadow** [reply]
looks like they don't support multi select yet

**Comment by JonahPlusPlus** [comment]
Windows + WSL
